api/base/views.py

```diff
@@ -75,3 +82,17 @@ def root(request, format=None):
             'users': absolute_reverse('users:user-list'),
         }
     })
+
+
+class OsfAPIViewMeta(type):
+    """Metaclass to ensure a base level of permissions are applied to all of our views
+    """
+    def __new__(cls, name, bases, attributes):
+        permission_classes = set((
+            drf_permissions.IsAuthenticatedOrReadOnly,
+            UserIsConfirmed,
+            UserIsNotDeactivated
+        ))
```

Comment:
Is there a view that doesn't have TokenHasScope, or do we expect one? I haven't looked at the scope permissions yet, but it looks like it's everywhere at first glance and so might make a good default permission class.